### ISSUE TITLE
Search Filter by the time of the connections 

### ISSUE THREAD

**Issue by programerr01** [tomquirk/linkedin-api#207]
`api.search_people(network_depth="S")`

this show me the list of secondary connections i have, i want to filter the data by the date

**Comment by abinpaul1** [comment]
I am not sure if the api provides such an option. Also I am not sure what you mean by date relating to a secondary connection. For first degree connections I believe the data is usually returned with latest connected person first.

**Comment by programerr01** [comment]
yes, i mean the date relating to the secondary connection. I have tried looking for pattern of date with the data returned but to me it appears it's rather random or sorted with other filter than by date

**Comment by abinpaul1** [comment]
I am still not clear what date relates to a secondary connection.
Date relating to a first degree connection is the date on which you got connected to that person.
But you are yet to get connected to a second degree connection, so what would date signify?

**Comment by programerr01** [comment]
in the api , i am also passing `connection_of` parameter. so, what i am looking for , is the data to be sorted by the date by which user which is passed to `connection_of` connected

**Comment by abinpaul1** [comment]
Oh okk... I might be wrong but I don't think that is supported
